liquibase-core/src/main/java/liquibase/parser/core/xml/XMLChangeLogSAXParser.java

```diff
@@ -150,6 +151,15 @@ public void fatalError(SAXParseException exception) throws SAXException {
             }
 
             throw new ChangeLogParseException("Invalid Migration File: " + reason, e);
+        } catch (InvalidPathException e) {
+            if (e.getInput().startsWith("s3://")) {
```

Comment:
Do you think we need this conditional? Could we throw this exception in all circumstances?